src/documents/components/ReviewDetail/ReviewDescription.js

```diff
@@ -0,0 +1,157 @@
+ */
+const ReviewDescription = ({reviewNode, updateReview}) => {
+  const [editing, setEditing] = useState(false);
+  const [editorState, setEditorState] = useState(
+    EditorState.createWithContent(
+      ContentState.createFromText(reviewNode.description || ''),
+    ),
+  );
+  const [editError, setEditError] = useState('');
+
+  const saveDescription = () => {
+    const rawState = convertToRaw(editorState.getCurrentContent());
+    const mdText = draftToMarkdown(rawState);
+    updateReview({
+      variables: {
+        id: reviewNode.id,
+        input: {
+          name: reviewNode.name,
+          description: mdText,
+        },
+      },
+    })
+      .then(() => setEditing(false))
+      .catch(err => setEditError(err.message));
+  };
+
+  return (
+    <Amplitude
+      eventProperties={inheritedProps => ({
+        ...inheritedProps,
+        scope: inheritedProps.scope
+          ? [...inheritedProps.scope, 'file description']
+          : ['file description'],
+      })}
+    >
+      <Header as="h4" color="grey">
+        Description
+        {updateReview && !editing && (
+          <Popup
+            content="Edit document description"
+            position="right center"
+            inverted
+            trigger={
+              <Button
+                size="mini"
+                labelPosition="left"
+                className="ml-15 text-primary"
+                onClick={() => setEditing(true)}
+              >
+                <Icon name="pencil" />
+                Edit
+              </Button>
+            }
+          />
+        )}
+      </Header>
+      {editing ? (
+        <>
+          <LogOnMount eventType="editor opened" />
+          <Segment>
+            <MarkdownEditor
+              editorState={editorState}
+              onEditorStateChange={e => setEditorState(e)}
+            />
+          </Segment>
+          <Segment basic clearing className="noMargin noPadding">
+            {editError && <span className="text-red">{editError}</span>}
+            <Amplitude
+              eventProperties={inheritedProps => ({
+                ...inheritedProps,
+                scope: inheritedProps.scope
+                  ? [...inheritedProps.scope, 'cancel button']
+                  : ['cancel button'],
+              })}
+            >
+              {({instrument}) => (
+                <Button
+                  floated="right"
+                  onClick={() => {
+                    setEditing(false);
+                  }}
+                >
+                  Cancel
+                </Button>
+              )}
+            </Amplitude>
+            <Amplitude
+              eventProperties={inheritedProps => ({
+                ...inheritedProps,
+                scope: inheritedProps.scope
+                  ? [...inheritedProps.scope, 'save button']
+                  : ['save button'],
+              })}
+            >
+              {({instrument}) => (
+                <Button
```

Comment:
It would be nice if this button was disabled and set to loading state while the mutation is being executed.